images/virtualization-artifact/pkg/common/datasource/ca_bundle.go

```diff
@@ -16,36 +16,94 @@ limitations under the License.
 
 package datasource
 
-import virtv2 "github.com/deckhouse/virtualization/api/core/v1alpha2"
+import (
+	"k8s.io/apimachinery/pkg/types"
+
+	virtv2 "github.com/deckhouse/virtualization/api/core/v1alpha2"
+)
 
 type CABundle struct {
 	Type           virtv2.DataSourceType
 	HTTP           *virtv2.DataSourceHTTP
-	ContainerImage *virtv2.DataSourceContainerRegistry
+	ContainerImage *DataSourceContainerRegistry
+}
+
+type DataSourceContainerRegistry struct {
```

Comment:
Name starts with the package name
Can be renamed to ContainerRegistry

Reply:
fixed